samples/build/build_buildpacks-v3-bundle.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Should this be under the test/data directory as this is not necessarily a good fit for our samples directory? Same applies to other files that you added.

Reply:
I figured those could be samples for reference. However, given that they differ only slightly from the Git base counterparts, I have no problem at all to remove them.